build.ps1

```diff
@@ -7,8 +7,7 @@ $PROD = "production"
 
 function Clone-Repo-And-Build {
     # Cloning the frontend repository.
-    chmod +x ./scripts/cloner.ps1
-    ./scripts/cloner.ps1 $BACKEND_REPOSITORY $args[0]
+    .\scripts\cloner.ps1 $BACKEND_REPOSITORY $args[0]
```

Comment:
Good call sinc the chmod operation does not exist in powershell.

Reply:
ok cambiaré eso

Reply:
nan mais sérieux je mets quoi ducoup ?

Reply:
Tu sauras, les commentaires ne sont pas nécessairement des trucs à changer.
Just take the compliment ;)

Reply:
mais ducoup pourquoi j'ai un request changes ?

Reply:
C'est les autres commentaires.